tests/ballerina-compiler-api-test/src/test/resources/test-src/expected-type/expression_test.bal

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2022 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+function add(int x, int y) returns int {
+    int sum = x + y;
+    // `return` statement returns a value.
+}
+
+function foo() {
+    int x = 1;
+    if (add(1,2) < )
+
+}
+
+function testFunc() {
+    int x = 1;
+    if (add(1,) < )
+
+}
+
+public function main() {
+    map<int> map1 = {field1: 10, field2: 20};
+    map<int> map2 = {field3:10, };
+}
+
+class Cl1 {
+    function init(int arg1, string arg2) {
+
```

Comment:
```suggestion
```
Superfluous newline. Check and update where applicable in other places as well. Also check and update the line positions in the test cases.